### ISSUE TITLE
ERROR: must provide pattern

### ISSUE THREAD

**Issue by waningflow** [sverweij/dependency-cruiser#123]
When the `pFileDir` is a absolute path which is same with `process.cwd()`，it will throw an error
```
ERROR: must provide pattern
```
### Expected Behavior
It should return the result.

### Current Behavior
It reports error.

### Possible Solution
In `src/main/filesAndDirs/normalize.js `, the `relativize` function will return empty, thus `glob.sync` receives an empty string.

### Steps to Reproduce (for bugs)
1. Install dependency-cruiser 
2. run 
```
depcruise --exclude "^node_modules" --output-type csv `pwd`
```

### Context
I'm using the api to get the dependency tree of a certain directory.

### Your Environment
* Version used: 4.15.3
* Node version: v11.13.0
* Operating System and version: Mac OS X 10.14.3
* Link to your project:

**Comment by sverweij** [comment]
@waningflow thanks! this should indeed not happen - I've reproduced the bug locally and have a fix that should be on npm tomorrow at the latest.

As a workaround until then you could use `.` instead of ``` `pwd` ```

B.t.w. Just curious: what's the reason you're using ``` `pwd` ``` instead of `.`?

**Comment by waningflow** [comment]
I use `` `pwd` `` just to reproduce this. In my project, I use an absolute path.

**Comment by sverweij** [comment]
@waningflow  I see - ... and one of the absolute paths happens to be the current working directory - clear ✔️ .

I've made a patch (see #124) and published it to npm with a beta tag: `dependency-cruiser@4.16.1-beta-1` - could you give it a try and let me know if it solves your issue? 

If it does I'll publish it tonight (GMT+1).

**Comment by waningflow** [comment]
If it passes the test, you can publish it directly, I think. Thank you~

Sander Verweij <notifications@github.com>于2019年4月28日 周日23:21写道：

**Comment by sverweij** [comment]
Thanks! 

I've published it to npm (`dependency-cruiser@4.16.1`) together with some external dependency bumps.
